walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc

```diff
@@ -54,8 +56,10 @@ image::images/arch.png[architecture, role="integr8ly-img-responsive"]
 [time=10]
 == Deploying the Data Sync showcase application
 
+//Home > Search > Select Templates from the drop down > Search for datasync and they should show up. Just make sure you're on the openshift namespace
```

Comment:
Should these `//` chars be left here?

Reply:
Yes, I made this as a comment for now based on some info from Jameel. Need to validate how this actually works in OS4 with the templates once we get the uxd cluster updated to the latest changes from the past sprint.